homeassistant/components/hue/sensor_base.py

```diff
@@ -47,16 +47,16 @@ class SensorManager:
     Intended to be a singleton.
     """
 
-    SCAN_INTERVAL = timedelta(seconds=5)
+    SCAN_INTERVAL = timedelta(seconds=0.1)
```

Comment:
absolutely not.

Reply:
This will flood the hub and force it to rate limit, resulting in intermittent failures. It will also use more resources on the Home Assistant host.